src/fsharp/NameResolution.fsi

```diff
@@ -244,6 +247,9 @@ val internal AddModuleOrNamespaceRefToNameEnv                    : TcGlobals ->
 /// Add a list of modules or namespaces to the name resolution environment
 val internal AddEntitiesContentsToNameEnv : TcGlobals -> ImportMap -> AccessorDomain -> range -> bool -> NameResolutionEnv -> ModuleOrNamespaceRef list -> NameResolutionEnv
```

Comment:
To be more accurate, we should rename `AddEntitiesContentsToNameEnv` to `AddModuleOrNamespacesContentsToNameEnv`.